cmd/utils.go

```diff
@@ -1003,14 +1003,15 @@ func getLatestVersion() string {
 
 func doVersionCheck(data []byte, old string, new string, file string) {
 	var latest = getLatestVersion()
-	if VERSION != "vlatest" && VERSION != latest {
+	if latest != "none" && VERSION != "vlatest" && VERSION != latest {
 		Info.log("*\n*\n*\n\nA new CLI update is available.\nPlease go to " + LatestVersionURL + " and update from " + VERSION + " --> " + latest + ".\n\n*\n*\n*")
 	}
 	output := bytes.Replace(data, []byte(old), []byte(new), -1)
```

Comment:
It would probably be better to use cliConfig set and write methods here.

Reply:
Done